ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -210,9 +214,11 @@ export default function CollectiblesItems({
                         }}
                       >
                         <img
+                          tabIndex={0}
```

Comment:
The parent element to this should be a `<button>` so we can avoid `tabIndex` and `keypress` shims to accomplish what `<button>` gives us.

Reply:
@darkwing Ive made this change, thanks!